.gitignore

```diff
@@ -7,4 +7,5 @@ gatling-results/
 /src/main/resources/install_cf_cli.sh
 /src/main/resources/manifest.yml
 performance-test-results/
-performance-test-results.zip
\ No newline at end of file
+performance-test-results.zip
+/run_performance_tests.sh
```

Comment:
run_performance_tests.sh is a script on my machine that's not ready to be committed (and isn't related to the current commit)